wp-html-mail/template.html

```diff
@@ -0,0 +1,183 @@
+
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+		<meta http-equiv="content-language" content="en">
+		<title>NHS Leadership Academy</title>
+		<meta content="width=device-width, initial-scale=1.0" name="viewport" />
+        <style type="text/css">
+			body{width:100% !important; -webkit-text-size-adjust:none; margin:0; padding:0;}
+			.ReadMsgBody{width:100%;}
+			.ExternalClass{width:100%; -webkit-text-size-adjust:none; -ms-text-size-adjust:none; margin:0; padding:0; background-color:#b9cfd3;}
+			.ExternalClass *{line-height:100%;}
+			.ExternalClass p{line-height: 100%;}
+  			p{margin:0; padding:0; margin-bottom:10px; font-size: 16px; line-height: 22px; font-family: Frutiger, Arial, sans-serif; color: #231F20; text-align: left;}
+			p a{color: #005EB8 !important; font-weight: bold !important; text-decoration: underline;}
+			img{outline:none; text-decoration:none; border:none;}
+			table td{border-collapse:collapse;}
+			.disable a{text-decoration:none; color:inherit;}
+			.unsubscribe a{text-decoration:none; color:#213F20 !important;}
+			.webversion a{text-decoration:underline; color:#213F20 !important;}
+			.footer a {font-size: 12px; !important;}
+			/*Responsive css*/
+			@media only screen and (max-width: 600px) {
+				*[class=gmailfix] {width:100% !important; min-width:1px !important;}
+				*[class=border] {width:90% !important; margin:0 auto; border-left: 1px solid #D8D6D6; border-right: 1px solid #D8D6D6;}
+				*[class=wrapper] {width:100% !important; margin:0 auto;}
+				*[class=inner] {width:90% !important; margin:0 auto; max-width: 560px !important;}
+				*[class=w100] {width:100% !important; margin:0 auto;}
+				*[class=img100] {width:100% !important; height:auto !important;}
```

Comment:
All classes below to `section-border-mid` seems to be not used in this email

Reply:
I've removed all unused classes